ftw/slacker/tests/__init__.py

```diff
@@ -0,0 +1,65 @@
+from contextlib import contextmanager
+from ftw.slacker.testing import FTW_SLACKER_FUNCTIONAL_TESTING
+from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
+from unittest2 import TestCase
+from ftw.slacker import slack_notifier
+import transaction
+import os
+
+
+class FunctionalTestCase(TestCase):
+
+    layer = FTW_SLACKER_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.request = self.layer['request']
+
+    def grant(self, *roles):
+        setRoles(self.portal, TEST_USER_ID, list(roles))
+        transaction.commit()
+
+
+class ResponseStub(object):
+
+    def __init__(self, status_code):
+        self.status_code = status_code
```

Comment:
I think you are not using `status_code`; could be removed from `ResponseStub ` and `RequestsMock` 🤔